03-Mains/Neutral_density.ipynb

```diff
@@ -6,14 +6,24 @@
    "metadata": {},
```

Comment:
<p>Do we need such a big celebration? (meaning, does it have to be a heading?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/627/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>